doc/man/salt.7

```diff
@@ -7571,9 +7571,14 @@ New in version 2014.7.0.
 Default: \fBsmart\fP
 .sp
 The pillar_source_merging_strategy option allows you to configure merging
-strategy between different sources. It accepts 4 values:
+strategy between different sources. It accepts 5 values:
```

Comment:
For future reference, you don't need to edit the man pages. They are automatically rebuilt on each release.